### DISCUSSION TITLE
VcOverlayHtml标签在viewer.zoomTo之后不会消失，还显示在屏幕上

### DISCUSSION THREAD

**Discussion by happychl** [zouyaoji/vue-cesium#424] [Q&A] [answered]
VcOverlayHtml标签显示在当前位置时正常，但是当点击按钮视角定位到其他地方时，标签不会消失，仍然显示在屏幕上，平移地图也不会改变位置，应该时bug吧

**Comment by zouyaoji** [comment]
平移也不会？在线文档是正常的呢

https://zouyaoji.top/vue-cesium/#/zh-CN/component/overlays/vc-overlay-html

**Reply by happychl** [reply]
可能我的描述过于抽象，基于这个在线demo，我做了一下调整，可以复现我的问题，点击跳转按钮应该就能看到了。
[Cesium Demo](https://codepen.io/willchen/pen/xxQvRmr)

**Reply by zouyaoji** [reply]
我看了，但还是没理解到你的意思。方便录个屏不。

**Comment by happychl** [comment]
![GIF 2023-8-14 13-31-07](https://github.com/zouyaoji/vue-cesium/assets/8717832/0c14ca5d-6afa-494d-a518-68c0addfb674)
一开始“成都欢迎您”这个标签是正常的，它有一个确定的经纬度，所以当你拖动地图时，它会随之移动。
但是点击跳转后，会定位到三维模型所在位置，这个位置和“成都欢迎您”这个标签的位置距离相差很多，不在一起，正常的表现是“成都欢迎您”这个标签应该不显示，但是从我的GIF图中可以看出仍然显示在屏幕上，并且拖动地图它的位置也不会变。

**Reply by zouyaoji** [reply]
表示重现不了，我用的你demo看也是正常的。。。你用新版谷歌浏览器试试？

**Reply by happychl** [reply]
![image](https://github.com/zouyaoji/vue-cesium/assets/8717832/6ee88005-075e-4a59-9253-4117b4ac29d4)
最新版chrome浏览器，在同事电脑上看也可以重现的。

**Reply by zouyaoji** [reply]
哦 明白了 要点那个跳转~  重现了哈。

**Comment by zouyaoji** [comment]


**Reply by zouyaoji** [reply, marked as answer]
已解决哈。zoomTo之后html在视野范围之外了，直接隐藏。下个版本修复。

**Reply by happychl** [reply]
好的，谢谢！
